docker/Dockerfile

```diff
@@ -19,14 +19,17 @@ FROM ubuntu:jammy AS base-runtime
 ARG AMD_ARCH=AMD_GFX90A
 ARG CUDA_ARCH=AMPERE80
 ARG CUDA_INSTALLER=https://developer.download.nvidia.com/compute/cuda/12.2.2/local_installers/cuda_12.2.2_535.104.05_linux.run
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Required to install `git` non-interactively (without choosing geographic regions, etc.)

Reply:
Why is this required now?

Reply:
Seems like moving git to that stage changed the behaviour of `apt`.